modules/getting-started/proc-add-source-control-rhdh-catalog.adoc

```diff
@@ -1,7 +1,7 @@
 [id='proc-add-source-control-rhdh-catalog_{context}']
 = Adding source control for Catalog in {product}
 
-To populate the Catalog in {product}, you need to add software templates, and to add the templates, you must enable a source control such as GitHub, GitLab, or BitBucket.
+To populate the Catalog in {product}, you need to add software templates, and to add the templates, you must enable a source control such as GitHub.
```

Comment:
```suggestion
To populate the Catalog in {product}, you need to add software templates, and to add the templates, you must enable a supported source control such as GitHub.
```
TBH, this is the only source control but hopefully we can expand the list in the future